cozy/cozy-mobile
# Fix realtime, fix version check on first start.

* Add test guide for human powered validation
* Fix realtime (fix files list update on changes)
* Fix messages on bad platform version during initialization process.
* Fix create photo folder, which were broken with the new routes.
* Add retries on initial replication get files and get folders steps, to hide view indexation.
* Fix check version algorithm
* Update min Proxy version
* Wait for data-system min version with https://github.com/cozy/cozy-data-system/commit/acdf87f46366351120a26e090f4610193c6ea7a6
